pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -151,27 +148,43 @@ def remove_initial_comment(self):
         except Exception as e:
             logging.exception(f"Failed to remove temp comments, error: {e}")
 
-    def publish_inline_comment(
-        self, comment: str, from_line: int, to_line: int, file: str
-    ):
-        payload = json.dumps(
-            {
-                "content": {
-                    "raw": comment,
-                },
-                "inline": {"to": from_line, "path": file},
-            }
-        )
+
+    # funtion to create_inline_comment
+    def create_inline_comment(self, body: str, relevant_file: str, relevant_line_in_file: str):
+        position, absolute_position = find_line_number_of_relevant_line_in_file(self.get_diff_files(), relevant_file.strip('`'), relevant_line_in_file)
+        if position == -1:
+            if get_settings().config.verbosity_level >= 2:
+                logging.info(f"Could not find position for {relevant_file} {relevant_line_in_file}")
+            subject_type = "FILE"
+        else:
+            subject_type = "LINE"
+        path = relevant_file.strip()
+        return dict(body=body, path=path, position=absolute_position) if subject_type == "LINE" else {}
+
+
+    def publish_inline_comment(self, comment: str, from_line: int, file: str):
+        payload = json.dumps( {
+            "content": {
+                "raw": comment,
+            },
+            "inline": {
+                "to": from_line,
+                "path": file
+            },
+        })
         response = requests.request(
             "POST", self.bitbucket_comment_api_url, data=payload, headers=self.headers
         )
         return response
```

Comment:
**Suggestion:** Consider using the `requests.post` method instead of `requests.request` with 'POST' as a parameter. This would make the code more readable and easier to understand.
```suggestion
            response = requests.post(self.bitbucket_comment_api_url, data=payload, headers=self.headers)
```